Source/Core/GeocoderService.js

```diff
@@ -0,0 +1,55 @@
+/*global define*/
+define([
+    './defineProperties',
+    './DeveloperError'
+    ], function(
+        defineProperties,
+        DeveloperError) {
+    'use strict';
+
+    /**
+     * @typedef {Object} GeocoderResult
+     * @property {String} displayName The display name for a location
+     * @property {Rectangle|Cartesian3} destination The bounding box for a location
+     */
+
+    /**
+     * Provides geocoding through an external service. This type describes an interface and
+     * is not intended to be used.
+     * @alias GeocoderService
+     * @constructor
+     *
+     * @see BingMapsGeocoderService
+     */
+    function GeocoderService () {
+        /**
+         * Indicates whether this geocoding service is to be used for autocomplete.
+         *
+         * @type {boolean}
+         * @default false
+         */
+        this.autoComplete = false;
+    }
+
+    defineProperties(GeocoderService.prototype, {
+        /**
+         * The name of this service to be displayed next to suggestions
+         * in case more than one geocoder is in use
+         * @type {String}
+         *
+         */
+        displayName : {
+            get : DeveloperError.throwInstantiationError
```

Comment:
This isn't actually being used at the moment, so perhaps I should remove it, but I thought it would be useful to have a name for each service for the case where somebody wanted to show to end users which results come from which geocoders. But that may be a premature consideration since it's not actually used... what do you think, should I just get rid of it?